### PR TITLE
Rename LLMonitor Callback Handler

### PR DESCRIPTION
**Pull request by hughcrt**
Following #14574, I've added `LunaryCallbackHandler` in `langchain_community` and kept `LLMonitorCallbackHandler` as an alias, in order to ensure backward compability.
Thanks!

@hwchase17

### CODE REVIEW

**Comment by hughcrt** [comment]
Hi @hwchase17, could you please check and confirm if the pull request can now be merged?

**Comment by vincelwt** [comment]
Hi @baskaryan pinging you also to see if there is something else to do to get this merged :) Thanks

File: libs/langchain/langchain/callbacks/llmonitor_callback.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 from langchain_community.callbacks.llmonitor_callback import (
     LLMonitorCallbackHandler,
+    identify,
```
Comment (hwchase17) [inline libs/langchain/langchain/callbacks/llmonitor_callback.py:3]:
```suggestion
```

File: libs/langchain/langchain/callbacks/llmonitor_callback.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
 )
 
-__all__ = [
-    "LLMonitorCallbackHandler",
-]
+__all__ = ["identify", "LLMonitorCallbackHandler"]
```
Comment (hwchase17) [inline libs/langchain/langchain/callbacks/llmonitor_callback.py:6]:
```suggestion
__all__ = ["LLMonitorCallbackHandler"]
```

**Review by hwchase17 — COMMENTED** [review]
left some minor comments just to make sure we dont update langchain, otherwise lgtm

**Comment by hughcrt** [comment]
Hi @hwchase17, I've made the changes you've suggested

**Comment by hwchase17** [comment]
hmm seems like some merge conflicts that need to be resolve via command line?

**Comment by hughcrt** [comment]
Hi @hwchase17, I've resolved the conflicts.

**Comment by efriis** [comment]
Changes a bunch of unrelated files (including CI) - feel free to clean up and reopen!

**Pull request closed without merging**